native/packages/react-native-bpk-component-switch/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-native-bpk-component-switch",
+  "version": "1.0.1",
```

Comment:
This should be `0.0.0`

Reply:
Argh! I copypasta'd this from the spinner component and I was so careful to scrub all mentions of it, then I missed this!